src/Models/Request/Point.php

```diff
@@ -8,18 +8,26 @@
 
 namespace Qdrant\Models\Request;
 
+use Qdrant\Models\MultiVectorStruct;
+use Qdrant\Models\VectorStruct;
+use Qdrant\Models\VectorStructInterface;
+
 class Point implements RequestModel
 {
     protected string $id;
-    protected array $vector;
+    protected VectorStructInterface $vector;
 
     /**
      * @var array|null Payload values (optional)
      */
     protected ?array $payload = null;
 
-    public function __construct(string $id, array $vector, array $payload = null)
+    public function __construct(string $id, VectorStructInterface|array $vector, array $payload = null)
```

Comment:
In my opinion, we can continue with an interface instead of an array, too.

Reply:
I mean that we can continue with interface directly. Without array.

Reply:
It's been a few weeks, so just confirming that you'd like me to remove `array` as an option for `$vector` in this constructor?

It would be neater removing `array`, but for backwards compatibility it does help. We could also mark array for removal in a future version and just trigger a warning when an array is passed for `$vector`.